### PR TITLE
Feat/set up repo

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Changes 
- set up. python repo 
- Set up lint & test in CI
- added required endpoints as per ticket 

### Ticket: 
- https://digicatapult.atlassian.net/browse/VR-88

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.12-slim
+LABEL MAINTAINER Digital Catapult
+# LABEL org.opencontainers.image.source=https://github.com/CDECatapult/soniclabs-resultsmanager
```
Comment (jonmattgray) [inline Dockerfile]: comment to delete

File: app/routes/posts.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+from typing import Any, List, Optional
+
+import httpx
+from fastapi import APIRouter, HTTPException
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+from uagents import Model
+
+veritableUrl = "http://localhost:3010"
+peerUrl = "http://localhost:3001/api"
```
Comment (jonmattgray) [inline app/routes/posts.py]: have as envs? example from other python repo https://github.com/CDECatapult/soniclabs-resultsmanager/blob/main/app/core/config.py

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+ticket: Ticket number for this bug (if applicable)
+---
+
+<!--
+
+Have you read nice-fetch-ai-adapter's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/nice-fetch-ai-adapter/.github/blob/main/CODE_OF_CONDUCT.md
```
Comment (dblane-digicatapult) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Have you read nice-fetch-ai-adapter's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/nice-fetch-ai-adapter/blob/main/CODE_OF_CONDUCT.md
```

File: .github/workflows/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+name: Feature
+about: What is this feature adding or changing?
+ticket: Ticket number for this feature (if applicable)
+---
+
+<!--
+
+Have you read nice-fetch-ai-adapter's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/nice-fetch-ai-adapter/.github/blob/main/CODE_OF_CONDUCT.md
```
Comment (dblane-digicatapult) [inline .github/workflows/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Have you read nice-fetch-ai-adapter's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/digicatapult/nice-fetch-ai-adapter/blob/main/CODE_OF_CONDUCT.md
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+
+- **Perform a [cursory search](https://github.com/search?q=+is%3Aissue+user%3Anice-fetch-ai-adapter)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on that repository and provide the following information by filling in [the template](https://github.com/digicatapult/nice-fetch-ai-adapter/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md).
```
Comment (dblane-digicatapult) [inline CONTRIBUTING.md]:
```suggestion
Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on that repository and provide the following information by filling in [the template](https://github.com/digicatapult/nice-fetch-ai-adapter/blob/main/.github/ISSUE_TEMPLATE/bug_report.md).
```

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by msterle — COMMENTED** [review]
lgtm

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**